modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -18,7 +18,19 @@ You can create an identity-based Identity and Access Management (IAM) policy tha
 . In the IAM console, select *Policies* from the left navigation menu.
 . Click *Create policy*.
 . Select the *JSON* tab to define the policy using JSON format.
-. Copy and paste the following JSON policy document into the editor:
+. To get the IP addresses that you need to enter into the JSON policy document, run the following command:
++
+[source,terminal]
+----
+$ ocm get /api/clusters_mgmt/v1/trusted_ip_addresses
+----
++
+[NOTE]
+====
+These IP addresses are not permanent and are subject to change. You must continuously review the API output and make the necessary updates in the JSON policy document.
+====
++
+. Copy the output IP addresses and add all the IP addresses into the `"aws:SourceIp": []` array in your following JSON policy document:
```

Comment:
Is it possible to add the file name after "policy document?"bThee's no identifiers that I can see in the review output.

Reply:
Hello @nalhadef . I have made the update as, "Copy the output IP addresses and add all the IP addresses into the `"aws:SourceIp": []` array in your `policy_document.json`:" and also updated Line 57. Let me know what do you think? Thanks!

Cc @wanghaoran1988 @yuwang-RH